openslides/core/migrations/0003_auto_20160803_1058.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.7 on 2016-08-03 10:58
+from __future__ import unicode_literals
+
+import uuid
+
+from django.db import migrations
+from openslides.core.models import Projector
+
+
+def add_speaker_overlay(apps, schema_editor):
+    """
+    Adds a list of speaker overlay to the existing projector id=1
+    """
+    projector = Projector.objects.get(id=1)
+    soverlay = {}
+    soverlay_uuid = uuid.uuid4().hex
+    soverlay[soverlay_uuid] = {
+        'name': 'core/speakeroverlay',
+        'stable' : True,
+        'uuid' : soverlay_uuid}
+    projector.elements.update(soverlay)
```

Comment:
After @normanjaeckel s comment I tried a different approach. I could produce a second projector with the speakeroverlay, but no clock. Now I'm trying to append the speakeroverlay to the dict of elements (see above). However, I am stuck again, with syntax I and the consulted web pages consider correct. As it doesn't work, my thinking appears to be wrong.